src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,69 @@
+.calendar {
+  width: $cell-size * 7 + ($margin-calendar * 2);
+  margin: 0 $margin-calendar;
+
+  &__head {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $between-col-gap;
+  }
+
+  &__day-name {
+    position: relative;
+    background-color: yellowgreen;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid #000;
+
+    @each $day, $index in $days {
+      &:nth-child(#{$index})::after {
+        content: "#{$day}";
+        position: absolute;
+        top: 32px;
```

Comment:
Use flexbox to center the day name inside the cell. Example: display: flex; align-items: center; justify-content: center;

Reply:
did